t/plugin/limit-count-redis-cluster3.t

```diff
@@ -77,3 +81,55 @@ __DATA__
     }
 --- response_body
 remaining: 1
+
+
+
+=== TEST 2: set route, with redis_cluster_nodes as environment variables and redis_cluster_name
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                ngx.HTTP_PUT,
+                [[{
+                    "uri": "/hello",
+                    "plugins": {
+                        "limit-count": {
+                            "count": 2,
+                            "time_window": 60,
+                            "rejected_code": 503,
+                            "key": "remote_addr",
+                            "policy": "redis-cluster",
+                            "redis_timeout": 1001,
+                            "redis_cluster_nodes": [
+                                "$ENV://REDIS_NODE_0",
+                                "$ENV://REDIS_NODE_1"
```

Comment:
I think you are going to encrypt the `redis_password`, am i wrong?

Reply:
it is not about encryption mostly, but about using environment variables or secrets to configure the plugin. Based on my changes each attribute of the plugin which is:
- string
- array of strings

can be configured via env variable or secret. 

So in this test i am checking if environment variables work on the example of array of strings field - redis_cluster_count.

Reply:
I misunderstood you, so you did two different thins in the pr: like `redis_password`